crates/stackable-operator/src/commons/authentication/mod.rs

```diff
@@ -77,6 +78,9 @@ pub enum AuthenticationClassProvider {
     /// The [TLS provider](DOCS_BASE_URL_PLACEHOLDER/concepts/authentication#_tls).
     /// The TLS AuthenticationClass is used when users should authenticate themselves with a TLS certificate.
     Tls(tls::AuthenticationProvider),
+
+    /// The Kerberos provider is used for Kerberos authentication and defines the secret used for generating keytabs.
```

Comment:
It would be great if you could add this provider to the concepts page and link them here (similar to TLS)

Reply:
Good idea. This should be available via https://github.com/stackabletech/documentation/pull/669

Reply:
See 03cd3f2

Reply:
Thanks! Approved